demo-app/app/build.gradle

```diff
@@ -41,7 +41,7 @@ dependencies {
     implementation 'androidx.core:core-ktx:1.3.0'
     implementation 'androidx.appcompat:appcompat:1.1.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
-    implementation 'com.banked:checkout:1.1.3'
+    implementation 'com.banked:checkout:1.1.4'
```

Comment:
can you bump this to `1.1.6`, i just pushed a couple of new versions to set checkout version headers 👍